lib/diaspora_federation/discovery/web_finger.rb

```diff
@@ -146,15 +165,38 @@ def to_s
         end
       end
 
+      def add_aliases_to(doc)
```

Comment:
I don't think we should do this, because as you wrote, it doesn't make much difference, but it breaks for example when you call `to_xml` twice on the same object.